extensions/rk35xx-panthor.sh

```diff
@@ -0,0 +1,24 @@
+#
```

Comment:
Why do we have 2 extensions? Can't rk-phantor do what rk35xx-panthor does?

Reply:
And / or cover Debian with the same extension. I just tested installing:
glmark2-wayland glmark2 glmark2-es2 glmark2-es2-wayland mesa-utils
on Bookworm and it works OOB. Which means oibaf couild be perhaps renamed into "3D" or "mesa" and cover both. This is also first step toward removing 3D from desktop app group.

Reply:
> And / or cover Debian with the same extension. I just tested installing: glmark2-wayland glmark2 glmark2-es2 glmark2-es2-wayland mesa-utils on Bookworm and it works OOB. Which means oibaf couild be perhaps renamed into "3D" or "mesa" and cover both. This is also first step toward removing 3D from desktop app group.

Yeah i know i didn't mean it. I just couldn't understand rk35xx-panthor.sh

Reply:
Well. I am trying to figure out what is the minimum extension count to cover all this. It is confusing to have that many. I am already lost ;)

Reply:
> Why do we have 2 extensions?

Because I messed up, badly. I had done rk35xx version and forgot to PR, then AF did his, then I tried to overwrite his and failed and ended up doing this mess, lol, let me fix.

Reply:
Fixed my mess.
Now it's a single thing, as it should (`rk-panthor`).
Should work exactly the same as AF's, except checks kernel and branch, and adds `-panthor` suffix to image filenames, so users know what it is.